docs/wxm-devices/helium-bundle/helium-install-weather-station.mdx

```diff
@@ -8,9 +8,10 @@ slug: /wxm-devices/helium-bundle/helium-install-weather-station
 import useBaseUrl from "@docusaurus/useBaseUrl";
 
 ## Prerequisites for proper installation
-1. Pole of at least 2 meters / 6.5ft (not included)
+1. Pole of at least 2m/6.5ft (not included)
 1. Base or bolts to secure in place (not included)
-1. Location and spacing (best practices for proper station installation)
+1. Location and spacing (<a href="/deployment-examples">Best practices for proper station installation</a>)
```

Comment:
The URL is wrong, please use this: https://docs.weatherxm.com/wxm-devices/helium-bundle/helium-install-weather-station#best-practices-for-proper-station-installation